.circleci/config.yml

```
@@ -36,12 +39,14 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
 
-      - run: cat /dev/null | sbt 'Test / compile'
+      # Compile, run tests, and produce coverage reports!
+      - run: cat /dev/null | sbt clean coverage test coverageReport coverageAggregate shutdown
```

If you want to close `stdin`, 

```suggestion
      - run: exec 0<&- && sbt clean coverage test coverageReport coverageAggregate shutdown
```

---

Neat! I just copied this from the template they posted. I like your solution better, but I would kinda prefer to stick to the template unless there is a good reason to deviate. Is there any advantages to this method?